nntrainer/tensor/weight.h

```diff
@@ -316,6 +317,16 @@ class Weight : public Var_Grad {
     return clip_by_global_norm > epsilon;
   }
 
+  /**
+   * @brief Check if the variable type is not full precision
+   *
+   * @return true if it is not full precsion
+   * @return false otherwise
+   */
+  bool isMixedPrecision() const {
+    return var->getDataType() == ml::train::TensorDim::DataType::FP32;
```

Comment:
I'm quite confused here. if this function returns true when the variable is **not** in full precision, shouldn't it be `!=` instead of `==`?

Reply:
fixed in #2566